### PR TITLE
Add GitHub action workflow for auto-assigning to main project

### PR DESCRIPTION
**Pull request by peterklingelhofer**
[GitHub Action: Assign to One Project](https://github.com/marketplace/actions/assign-to-one-project#:~:text=GitHub%20Action%20for%20Assign%20to,columns%20in%20your%20project%20dashboard)

@simonv3, you may have to do this part: 
> Generate a token from the Organisation settings or User Settings and add it as a secret in the repository secrets as `MY_GITHUB_TOKEN`

Including this for reference as well: [GitHub Actions Quickstart](https://docs.github.com/en/actions/quickstart)

### CODE REVIEW

File: .github/workflows/github-actions.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+jobs:
+  assign_one_project:
+    runs-on: ubuntu-latest
+    name: Assign to One Project
+    steps:
+    - name: Assign NEW issues and NEW pull requests to project 1
+      uses: simonv3/assign-one-project-github-action@1.3.1
```
Comment (simonv3) [inline .github/workflows/github-actions.yaml:19]: This should be the original name of the action 
```suggestion
      uses: srggrs/assign-one-project-github-action@1.2.1
```

**Comment by simonv3** [comment]
@peterklingelhofer can you try the above fix? I already have a `GITHUB_TOKEN` so it should be fine since that's what you're pointing at (the other actions use it too). 

Looks like the linter and cypress tests ran on this! That's cool. 

The lint commit message action failed because your commit message doesn't follow commitlint, but that's fine. I'd like to see if the action passes when pointing to the right place.

**Comment by peterklingelhofer** [comment]
@simonv3 Sorry, forgot about the commit message again. I wish I could use SmartGit to edit the commit message after pushing but the only thing that will let me push to my fork for some reason is GitHub Desktop (which I typically never use). Will deal with it when I get the time.

That's great the jobs ran this time though!

**Comment by simonv3** [comment]
^ well that's promising!

**Pull request merged**